<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Priorities</class>
 <widget class="QWidget" name="Priorities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The priopities section defines how to compare schedules. &lt;/p&gt;&lt;p&gt;It is used to calculate the score of a schedule that is listed in the &lt;span style=&quot; font-style:italic;&quot;&gt;simulation_summary.txt&lt;/span&gt; file. &lt;/p&gt;&lt;p&gt;Additonally, it defines which schedule is reported to be the &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;recommended&lt;/span&gt;&amp;quot; schedule in the log file. The &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;recommended&lt;/span&gt;&amp;quot; schedule is selected based on the mean formal error or repeatability. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The reference quantile is used to define how the scores are calculated. &lt;/p&gt;&lt;p&gt;The score is calculated based on individual scores per parameters:&lt;/p&gt;&lt;p&gt;For a given parameter, all simulated accuracies are compared. The highest accuracy gets a score of 1. Accuracy at the reference quantile of all accuracies and poorer get a score of 0. Linear interpolation is used in between. &lt;br/&gt;&lt;/p&gt;&lt;p&gt;Use the spin box to define how much weight you want to give one parameter. &lt;/p&gt;&lt;p&gt;e.g.: if you are especially interested in dUT1 give this parameter a high weight. &lt;/p&gt;&lt;p&gt;It is advisable to always select a high number of observations as a goal of scheduling to minimize inaccuracies from the simulations. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>recommend based on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_noRec">
       <property name="text">
        <string>no recommendation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_mfe">
       <property name="text">
        <string>mean formal error</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_rep">
       <property name="text">
        <string>repeatability</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>reference quantile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_quantile">
       <property name="minimum">
        <double>0.050000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.750000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget_params">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>priority</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <item>
      <property name="text">
       <string>#obs</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EOP</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>XPO</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YPO</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dUT1</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NUTX</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NUTY</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>stations</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
